dart/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ description: >
 homepage: https://github.com/getsentry/sentry-dart
 
 environment:
-  sdk: ^2.0.0
+  sdk: ^2.1.0
```

Comment:
why this change?

Reply:
@rxlabz mind explaining?

Reply:
an autofix options error : clicked on 'upgrade sdk' instead of 'import dart:async', and didn't realized